internal/controllers/core/clustercatalog_controller.go

```diff
@@ -349,7 +352,10 @@ func updateStatusCatalogDisabled(status *v1alpha1.ClusterCatalogStatus, generati
 
 func updateStatusNotServing(status *v1alpha1.ClusterCatalogStatus, generation int64) {
 	status.ResolvedSource = nil
-	status.ContentURL = ""
+	if status.URLs == nil {
+		status.URLs = &v1alpha1.CatalogURLs{}
+	}
+	status.URLs.Base = ""
```

Comment:
I think we can just set `status.URLs = nil`, right?
```suggestion
	status.URLs = nil
```